jsapp/xlform/src/model.row.coffee

```diff
@@ -283,6 +301,7 @@ module.exports = do ->
       output['kobo--score-choices'] = @getList().get('name')
       survey_arr.push(output)
       ``
+      return
```

Comment:
Same question here - shouldn't we remove the backtics? Maybe two backticks were being used as implicit return?